<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRawFsdMessagesComponent</class>
 <widget class="QFrame" name="CRawFsdMessagesComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log component</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QListWidget" name="lw_RawFsdMessages">
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="gb_WriteToFile">
     <property name="title">
      <string>Write to File</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_FileDir">
        <property name="text">
         <string>Directory:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cb_FileWritingMode">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_FileWriteMode">
        <property name="text">
         <string>Write Mode</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="cb_EnableFileWriting">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="le_FileDir">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pb_SelectFileDir">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Set</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
